velocity/src/main/java/org/geysermc/floodgate/listener/VelocityListener.java

```diff
@@ -69,26 +70,21 @@ public final class VelocityListener {
         INITIAL_MINECRAFT_CONNECTION = getFieldOfType(initialConnection, minecraftConnection);
 
         // Since Velocity 3.1.0
-        Class<?> loginInboundConnection =
```

Comment:
Ideally please remove the formatting changes so the diff is clean and only contains actual code changes.

Reply:
My bad, must've done this while getting rid of imports that weren't used*